<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pingButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="serialPortLabel">
    <property name="text">
     <string>Puerto Serie:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>64</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>14</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>771</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QCheckBox" name="RTIcheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>RTI Botones</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="Led" name="SW1">
      <property name="toolTip">
       <string>Color Led component</string>
      </property>
      <property name="whatsThis">
       <string>Led indicator</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>SW1</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="Led" name="SW2">
      <property name="toolTip">
       <string>Color Led component</string>
      </property>
      <property name="whatsThis">
       <string>Led indicator</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SW2</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="switchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Detectar Botones</string>
     </property>
    </widget>
    <widget class="QComboBox" name="modeComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>97</width>
       <height>66</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="rojo">
      <property name="text">
       <string>Rojo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="verde">
      <property name="text">
       <string>Verde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="azul">
      <property name="text">
       <string>Azul</string>
      </property>
     </widget>
    </widget>
    <widget class="QwtKnob" name="Knob">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>129</width>
       <height>134</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="singleSteps">
      <UInt>0</UInt>
     </property>
     <property name="pageSteps">
      <UInt>10</UInt>
     </property>
    </widget>
    <widget class="color_widgets::ColorWheel" name="colorWheel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QCheckBox" name="monitorCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Activar monitorizaci√≥n</string>
     </property>
    </widget>
    <widget class="Led" name="AIN0Led">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="AIN1Led">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="AIN3Led">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="AIN2Led">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="QLabel" name="AIN0Label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AIN0</string>
     </property>
    </widget>
    <widget class="QLabel" name="AIN1Label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AIN1</string>
     </property>
    </widget>
    <widget class="QLabel" name="AIN2Label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AIN2</string>
     </property>
    </widget>
    <widget class="QLabel" name="AIN3Label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AIN3</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="AIN0lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="AIN0Knob">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>1.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
     <property name="totalSteps">
      <UInt>8</UInt>
     </property>
    </widget>
    <widget class="QLCDNumber" name="AIN1lcdNumber">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="AIN1Knob">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>1.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>2.500000000000000</double>
     </property>
     <property name="totalSteps">
      <UInt>8</UInt>
     </property>
    </widget>
    <widget class="QLCDNumber" name="AIN2lcdNumber">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="AIN2Knob">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>1.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>3.500000000000000</double>
     </property>
     <property name="totalSteps">
      <UInt>8</UInt>
     </property>
    </widget>
    <widget class="QLCDNumber" name="AIN3lcdNumber">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="AIN3Knob">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>1.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="scaleStepSize">
      <double>0.000000000000000</double>
     </property>
     <property name="value">
      <double>4.500000000000000</double>
     </property>
     <property name="totalSteps">
      <UInt>8</UInt>
     </property>
    </widget>
    <widget class="Led" name="PB0Led">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="PB1Led">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="PB2Led">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="Led" name="PB3Led">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PB0CheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PB0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PB1Checkbox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PB1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PB2Checkbox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PB2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PB3Checkbox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PB3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QCheckBox" name="oscCheckBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adquisici√≥n</string>
     </property>
    </widget>
    <widget class="QwtPlot" name="grafica">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>400</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="frecSpinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>25000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hz</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="escalaSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Escala</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="hourPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar hora</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>H:mm:ss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="receiveHourPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recibir hora</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
